modules/mcuboot/CMakeLists.txt

```diff
@@ -823,6 +897,12 @@ CONFIG_DFU_MULTI_IMAGE_PACKAGE_BUILD cannot be used with CONFIG_XIP_SPLIT_IMAGE.
     list(APPEND dfu_multi_image_targets signed_s0_target signed_s1_target)
   endif()
 
+  if(CONFIG_NRF_WIFI_FW_PATCH_DFU)
+    list(APPEND dfu_multi_image_ids 2)
```

Comment:
Should this match the image number? @de-nordic

Reply:
I've changed it, Is it correct now?